lib/screenplay_web/auth_manager/auth_manager.ex

```diff
@@ -3,10 +3,12 @@ defmodule ScreenplayWeb.AuthManager do
 
   use Guardian, otp_app: :screenplay
 
-  @type access_level :: :none | :read_only | :emergency_admin | :screens_config_admmin
+  @type access_level ::
+          :none | :read_only | :emergency_admin | :screens_config_admin | :pa_message_admin
 
-  @screenplay_admin_role "screenplay-emergency-admin"
+  @screenplay_emergency_admin_role "screenplay-emergency-admin"
   @screens_admin "screens-admin"
```

Comment:
What's the distinction between an admin and an emergency-admin? I notice in our fake dev permissions, we only grant the "emergency" one.

Reply:
`emergency-admin` is the permission needed to access the takeover tool. `screens-admin` will be used to access the new Permanent Configuration tool (still in a feature branch). I'm not 100% sure why it is in this file already considering there is a feature branch for that. It must have been done before realizing we should be merging to that branch instead. It will be added to the fake permissions when that branch is merged into main.